### PR TITLE
Update voice-tone-and-writing-style-guidelines.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated writing guidelines to include family-friendly. Reorganized list slightly.

### CODE REVIEW

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ Mattermost's voice is:
 To achieve this, we keep the following principles in mind when writing content:
 
 * Write clearly and concisely - get to the point quickly without losing valuable information
-* Use simple, plain language that is easy to understand and avoid jargon
+* Use simple, plain language that is easy to understand and family-friendly
```
Comment (cwarnermm) [inline operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md:22]: Oh, I like this addition of "family-friendly"!

**Review by cwarnermm — APPROVED** [review]
Great updates!

**Review by JustinReynolds-MM — APPROVED** [review]
Looks good to me!

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 1 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md:22]: I can't take credit for it, but it's a great addition. :)

**Pull request merged**